fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/logical/InApplyToJoin.java

```diff
@@ -51,15 +56,19 @@ public Rule build() {
                 predicate = new EqualTo(((InSubquery) apply.getSubqueryExpr()).getCompareExpr(),
                         apply.right().getOutput().get(0));
             }
-
+            List<Expression> conjuncts = ExpressionUtils.extractConjunction(predicate);
+            if (conjuncts.stream().anyMatch(expression -> expression.children().stream()
+                    .anyMatch(expr -> expr.getDataType() == BitmapType.INSTANCE))) {
+                throw new NereidsException(new AnalysisException("nereids don't support bitmap filter"));
```

Comment:
add a TODO on it, we will fix it later.
```suggestion
                throw new AnalysisException("nereids don't support bitmap filter");
```